dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi

```diff
@@ -319,6 +331,16 @@
 			compatible = "intel,adsp-shim";
 			reg = <0x71f00 0x100>;
 		};
+		
+		dfpmcch: dfpmcch@71ac0 {
```

Comment:
`dfpmcch` covers `lps` memory mapping. Consider making `lps` a part of `dfpmcch` or replacing it completely

Reply:
+1

Reply:
Already merged:
https://github.com/zephyrproject-rtos/zephyr/pull/52343